ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/StateMachine.hs

```diff
@@ -1698,16 +1728,16 @@ traceEventName = \case
     TraceOpenEvent              ev    -> "Open."              <> constrName ev
     TraceGCEvent                ev    -> "GC."                <> constrName ev
     TraceIteratorEvent          ev    -> "Iterator."          <> constrName ev
-    TraceSnapshotEvent          ev    -> "Ledger."            <> constrName ev
-    TraceLedgerReplayEvent      ev    -> "LedgerReplay."      <> constrName ev
+    TraceLedgerDBEvent          ev    -> "Ledger."            <> constrName ev
+--    TraceLedgerReplayEvent      ev    -> "LedgerReplay."      <> constrName ev
```

Comment:
Shouldn't `TraceLedgerReplayEvent` this be removed instead of being commented out? It is now part of `TraceLedgerDBEvent`.